src/main/java/com/github/devotedmc/hiddenore/Config.java

```diff
@@ -308,15 +346,15 @@ public static String getPrettyName(String name, short durability) {
 	public static File getTrackFile() {
 		return trackFile;
 	}
-	
+
 	public static int getTransformAttemptMultiplier() {
 		return instance.transformAttemptMultiplier;
 	}
-	
-	public ConfigurationSection getWorldGenerations() {
-		return file.getConfigurationSection("clear_ores");
+
+	public ConfigurationSection getWorldGenerations(String worldName) {
+		return configurations.get(worldName).getConfigurationSection("clear_ores");
```

Comment:
A little bit of defensive coding here might be good, namely, use `configurations.getOrDefault(worldName, mainConfig).getConfigurationSection("clear_ores");` so if _somehow_ an improper world definition is provided, we aren't attempting to use an accessor on a null object.